docs/graphql/manual/mutations/insert.rst

```diff
@@ -241,33 +247,49 @@ in the response
         affected_rows
         returning {
           id
-          title
-          author {
-            id
-            name
+          name
+          address {
```

Comment:
can we also get the auto-generated `id`s, `address_id` and ``author_id`` fields in the response so that its clear what values have been auto-generated and set